### ISSUE TITLE
Compatibility  Layer and update vapor to LOVE 0.9.2

### ISSUE THREAD

**Issue by Bobbyjoness** [josefnpat/vapor#188]
I think creating a compatibility layer would be the best way for getting vapor to update to 0.9.2 and future versions. I myself may try to do this if its not too much of a daunting task.

**Comment by josefnpat** [comment]
This issue is rather hand in hand. I'm ok with this as long as it's a stop-gap solution.

We may run into issues if 0.10.0 is released and there is no compatibility layer for 0.8.0 -> 0.10.0 and/or 0.9.x -> 0.10.0

**Comment by Bobbyjoness** [comment]
Hey so my compatibility layer is almost complete and most games run. You can see my work here https://github.com/Bobbyjoness/vapor/tree/addCompat . It is mostly complete but I can use help testing games. Also I added a check to see if the game is 0.8.0 before adding compat. So far there is a couple issues that can't be fixed and no its not threads. Those are actually working :) lol

**Comment by josefnpat** [comment]
https://github.com/josefnpat/vapor/pull/199

**Comment by Bobbyjoness** [comment]
Can we open submission on the website? I want to have some 0.9.x games in vapor for testing.

**Comment by josefnpat** [comment]
I quickly checked the website, and currently it only supports 0.8.0 games.

I will have some time soon (start of next month) to extend it to support 0.9.[0|1|2] frameworks on the api and I will make a new endpoint so that you can test your branch.

If this is agreeable, please hit me up here at the start of the next month, and we can work this out!

**Comment by pablomayobre** [comment]
With endpoint you mean a different `game.json`? because there should still be a pure 0.8.0 `game.json` for the old version

**Comment by josefnpat** [comment]
Yup, it's dynamically generated here: http://vapor.love2d.org/games.json
